### PR TITLE
Link to https://k8s.dev/ docs about good first issues

### PR DESCRIPTION
**Pull request by sftim**
Where we mention the "help" or "good first issue" labels, let's hyperlink to https://kubernetes.dev/docs/guide/help-wanted/ (our
official page on that topic).

The specific page to link to is https://kubernetes.dev/docs/guide/help-wanted/

### CODE REVIEW

File: content/en/docs/contribute/participate/pr-wranglers.md [hunk 1]
```diff
@@ -29,7 +29,13 @@ Each day in a week-long shift as PR Wrangler:
     - You can also tag a [SIG](https://github.com/kubernetes/community/blob/master/sig-list.md) for a review by commenting `@kubernetes/<sig>-pr-reviews` on the PR.
 - Use the `/approve` comment to approve a PR for merging. Merge the PR when ready.
     - PRs should have a `/lgtm` comment from another member before merging.
-    - Consider accepting technically accurate content that doesn't meet the [style guidelines](/docs/contribute/style/style-guide/). Open a new issue with the label `good first issue` to address style concerns.
+    - Consider accepting technically accurate content that doesn't meet the
+      [style guidelines](/docs/contribute/style/style-guide/). As you approve the change,
+      open a new issue to address the style concern. You can usually write these style fix
+      issues as [good first issues](https://kubernetes.dev/docs/guide/help-wanted/#good-first-issue).
+    - Using style fixups as good first issues is a good way to ensure a supply of easier tasks
+      to help onboard new contributors.
```
Comment (sftim) [inline content/en/docs/contribute/participate/pr-wranglers.md:37]: I reworded this a bit, too.

**Comment by divya-mohan0209** [comment]
@sftim would it be beneficial to link to the GitHub link for k/website issues filtered with the required labels somewhere?

**Comment by sftim** [comment]
> would it be beneficial to link to the GitHub link for k/website issues filtered with the required labels somewhere?

I'm not sure what you're suggesting, but I doubt I'd object.

**Comment by jimangel** [comment]
/lgtm
/approve

(still open to @divya-mohan0209's feedback, but merging PR for velocity; it sounds like the follow-up might be a `good-first-issue`?)

**Pull request merged**